build_tools/customize_build.py

```diff
@@ -0,0 +1,78 @@
+import os
+import tempfile
+
+import requests
+from kolibri.utils import conf
+
+os.environ.setdefault(
+    "BUILD_TIME_PLUGINS", os.path.join(os.path.dirname(__file__), "default_plugins.txt")
```

Comment:
I believe it's best-practice to prefix environment variables with the application name